DOCS-10137 Adding note for known issue (#1282)
mulesoft/docs-connectors: salesforce/10.9/modules/ROOT/pages/salesforce-connector-processing-events.adoc

For each message that comes through a topic or streaming channel to which the connector is subscribed, the connector updates the object store information using up to four transactions.

[NOTE]

> ---

Reviewer: (Oops...a CU edit...):
This is probably my confusion in seeing this out of context, but I am wondering if topic/streaming channel (above), multiple workers, and multiple instances all refer to the same thing: is a worker also an instance or an app?

I'm looking for a simpler wording around these points:

* You have multiple xyz
* Each xyz will process each message, creating overhead
* Avoid redundant processing by...
** filter duplicates
** have one app "deliver the mail"?

> If you have multiple workers deployed in CloudHub, each message is processed by all of the Mule instances when using the *Replay Topic* or *Replay Channel* operations.  

Reviewer 2: ```suggestion
If you have multiple workers deployed in CloudHub, all Mule instances process each message when using the *Replay Topic* or *Replay Channel* operations.  
```

> If you have multiple workers deployed in CloudHub, the same messages are processed by all instances of the app when using the *Replay Topic* or *Replay Channel* operations.  

Reviewer: ```suggestion
If you have multiple workers deployed in CloudHub, each message is processed by all instances of the app when using the *Replay Topic* or *Replay Channel* operations.  
```

> To avoid processing the same messages across multiple instances, you can do one of the following:

Reviewer: ```suggestion
To avoid processing one message across multiple instances, you can do one of the following:
```

> To avoid duplicate processing of the same message across multiple Mule instances, you can do one of the following:

Reviewer 2: ```suggestion
To avoid multiple Mule instances processing the same message, you can do one of the following:
```

* Modify your app to filter out duplicate messages.

> * Designate a single app as a receiver and then use a shared queue to send messages to different apps and workers to process.

Reviewer: How does a shared queue prevent duplicate effort? Will your audience understand?

> ---

Reviewer 3: "to process" should only appear once
